docs/user/arena-setup.rst

```diff
@@ -2,269 +2,272 @@
++-------------------+-------------------------------------+------------+
+| show              | Show a region, warp, spawnpoint(s), | N/A        |
+| ``[<region>|``    | or container as red wool blocks.    |            |
+| ``<warp>|``       | *Example*: ``show sp``              |            |
+| ``<spawnpoint>|`` |                                     |            |
+| ``<container>]``  |                                     |            |
++-------------------+-------------------------------------+------------+
+
+- **Valid regions**: ``ar`` (arena region), ``lr`` (lobby region)
+- **Valid amounts**: Any positive integer (i.e. whole number)
+- **Valid directions**: ``up``, ``down``, ``out``
+- **Valid warps**: ``arena``, ``lobby``, ``spec``, ``exit``
+- **Valid spawnpoints**: ``spawns`` (or ``sp``)
+- **Valid containers**: ``chests`` (or ``c``)
+
+Toolbox
+=======
+
+The Toolbox is a set of golden tools. Each tool has a specific functions. We use
+them to set up regions, warps, spawnpoints, and containers. Toolbox tools are
+used with either a left- or right-click.
+
+Tool functions are also described in the *item tooltip* in your inventory.
+
+Region tools
+------------
+
+|r-icon|
+
+Arena and lobby regions are defined with Region tools (golden axes). There are
+two golden axes in the Toolbox. One is for *arena setup* and the other is for
+*lobby setup*. The tools are named accordingly.
+
+Region tools behave similarly to the WorldEdit wand (wooden axe). If you are
+familiar with regions in WorldEdit, Region tools should feel familiar.
+
+- **Left-click**: Sets first point on clicked block
+- **Right-click**: Sets second point on clicked block
+
+When both points are set, the region is defined. ``show ar`` (or ``show lr``)
+lets you check the region spans the desired area. If the region is too small,
+use the ``expand`` command (see above) to make it bigger.
+
+The region must be three-dimensional (like a box) and not two-dimensional (flat
+rectangle). Make sure your arena floor is contained in the region selection
+(expanding a block or two below the floor is recommended).
+
+Warp tool
+---------
+
+|w-icon|
+
+All warps are defined using the Warp tool (golden hoe). The tool defines any of
+the four types of warps depending which one is selected.
+
+- **Left-click:** Set selected warp type on top of clicked block
+- **Right-click:** Cycle between warp types
+
+A selected warp is placed on top of the clicked block. The direction you are
+looking is also taken into account.
+
+Arena, lobby, and spectator warps are required. An exit warp is optional.
+
+Spawnpoint tool 
+---------------
+
+|s-icon|
+
+Spawnpoints are set up with the Spawnpoint tool (golden sword). The tool allows
+an administrator to set or remove spawnpoints for mobs.
+
+- **Left-click:** Add spawnpoint on top of clicked block
+- **Right-click:** Remove spawnpoint on top of clicked block (if one exists)
+
+Many spawnpoints recommended
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I don't think this "headline" is necessary. It looks a bit odd (at least on github) and provides little more value than the paragraph following it. I think the paragraph with the emphasis is perfectly fine :)

Reply:
In retrospect, I think I was a little header-happy here. I think it's also not that valuable and will remove it shortly.